src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -209,6 +213,40 @@ public IndexFeatureSet indexFeatureSet()
         return indexFeatureSet;
     }
 
+    /**
+     * Returns whether this query is selection the {@link PrimaryKey}.
```

Comment:
I don't understand this sentence. Did you mean `selection of` or `selecting`?

Reply:
I meant `selecting`, I'll change it.